integration/test/README.md

```diff
@@ -44,19 +26,37 @@ application to wrap target job execution.
               $PATH_TO_DDT_CLIENT_SESSION_FILE" \
               ./geopm_test_integration.py -v TestIntegration.test_count`
 
-EXAMPLES
---------
-Since the log files that are emitted from the individual tests are
-opened and appended to on subsequent runs, it may become necessary to
-periodically clean up the files to avoid conflating experimental data.
-```git clean``` can be used in this instance to clean up all of the
-temporary files:
-
-    $ git clean -fd .
-    $ export GEOPM_RUN_LONG_TESTS=true
-    $ ./geopm_test_integration.py -v TestIntegration.test_power_consumption
-
-Or as a one-liner:
-
-    $ git clean -fd . ; GEOPM_RUN_LONG_TESTS=true ./geopm_test_integration.py -v TestIntegration.test_power_consumption
-
+EXAMPLE
+-------
+
+```
+#!/bin/bash
+
+# Set to geopm source code location
+export GEOPM_SOURCE=/home/test/Git/geopm
+
+# Set to the install prefix provided to configure
+export GEOPM_INSTALL=/home/test/build/geopm
+
+# Find location of geopmdpy site-packages
+export GEOPM_PYTHONPATH=$(dirname $(find ${GEOPM_INSTALL} -name geopmdpy -type d))
+
+# Set PATH values
+export LD_LIBRARY_PATH=${GEOPM_INSTALL}/lib:${LD_LIBRARY_PATH}
+export PYTHONPATH=${GEOPM_PYTHONPATH}:${PYTHONPATH}
+export PATH=${GEOPM_INSTALL}/bin:${PATH}
+
+python3 -m unittest discover \
+        --top-level-directory ${GEOPM_SOURCE} \
+        --start-directory ${GEOPM_SOURCE}/integration/test \
+        --pattern 'test_*.py' \
```

Comment:
According to the [docs](https://docs.python.org/3/library/unittest.html#unittest-test-discovery), the default pattern is `test*.py` when it is not specified.

Is there a reason that is not sufficient?

Reply:
No, we could omit line 52.